src/System.IO.FileSystem/tests/File/Move.cs

```diff
@@ -16,6 +16,25 @@ public virtual void Move(string sourceFile, string destFile)
             File.Move(sourceFile, destFile);
         }
 
+        public virtual void Move(string sourceFile, string destFile, bool overwrite)
```

Comment:
You need to make this class partial and create a `.netcoreapp.cs` version of it and put it in the proper item group in the .csproj. We use the same tests against .NET Framework (4.x) to ensure we understand correctness. As this is a new API we can only compile against netcoreapp.

Reply:
Erp, sorry. Crossing my streams. That is what you'd do if this was new. You need to conditionalize the tests as the behavior changed. I'll give you some links.

Reply:
Argh- don't try and review code with 4 teenagers interupting you. :)